P5/webnav/popupFootnotes.js

```diff
@@ -30,13 +30,24 @@ document.addEventListener("DOMContentLoaded", function() {
   for (var i=0; i<links.length; i++){
     if (links[i].hasAttribute("href") && links[i].getAttribute('href').substring(0, 5) == '#Note'){
       if (links[i].getAttribute('class') != 'link_return'){
-        var dest = links[i].getAttribute('href').substring(1);
+        /* When an anchor link is clicked, we use its @href value to determine which 
+         note's content should be loaded into the popup. */
         links[i].onclick = function (e) {
```

Comment:
I know this is from the original, but if we're changing this, then why not move more toward ES6

```js
const links = document.querySelectorAll('a[href ^= "#Note"]:not('.link_return')');
links.forEach(link => {
  link.addEventListener('click', (e) => {
     e.preventDefault();
   // And then the rest as @amclark42 proposed
  })
}
```